### PR TITLE
Feat/styles

### PR DESCRIPTION
**Pull request by katerinamartysko**

### CODE REVIEW

**Comment by katerinamartysko** [comment]
[Support abbreviation avatars if it doesn't exist](https://trello.com/c/9pt1HUGH/16-support-abbreviation-avatars-if-it-doesnt-exist)

File: src/components/ClientId.tsx [hunk 1]
```diff
@@ -1,82 +1,104 @@
...
-      </div>
 
-      {clientIdError && <h1 className="error">Произошла ошибка {clientIdError}</h1>}
-      {isClientIdLoading && (
-        <Box sx={{ display: 'flex' }}>
-          <CircularProgress />
-        </Box>
-      )}
-    </div>
+        {clientIdError && <h1 className="error">Произошла ошибка {clientIdError}</h1>}
```
Comment (LexaLukovka) [inline src/components/ClientId.tsx]: Please use `Typography` variant h1

File: src/components/ClientsListContainer.tsx [hunk 2]
```diff
@@ -49,11 +69,13 @@ export const ClientsListContainer: FC<Props> = ({ clientID, onSetClientID }) =>
     searchClients(search);
   };
 
+  if (client && isHideClientsList) return null;
   if (isClientsLoading) return <CircularProgress className={classes.load} />;
   if (clientError) return <h1 className="error">Произошла ошибка {clientError}</h1>;
```
Comment (LexaLukovka) [inline src/components/ClientsListContainer.tsx]: Please use `Typography`

File: src/components/Search.tsx [hunk 3]
```diff
@@ -6,6 +6,21 @@ import { theme } from '../utils';
...
+    [theme.breakpoints.up('xs')]: {
+      width: 'calc(100% - 20px)',
+    },
+    [theme.breakpoints.up('sm')]: {
+      width: '205px',
+    },
+    [theme.breakpoints.up('md')]: {
+      width: '215px',
+    },
+    [theme.breakpoints.up('lg')]: {
```
Comment (LexaLukovka) [inline src/components/Search.tsx:18]: remove unnecessary styles

File: src/utils/getFirstLetters.ts [hunk 4]
```diff
@@ -1,6 +1,5 @@
 import { General } from '../api/types';
 
 export const getFirstLetters = (general: General): string => {
```
Comment (LexaLukovka) [inline src/utils/getFirstLetters.ts]: Please receive object with first and last name not a `general: General`

**Review by LexaLukovka — CHANGES_REQUESTED** [review]

File: src/components/ClientId.tsx (hunk 1 above)
Comment (katerinamartysko) [reply src/components/ClientId.tsx]: Done

File: src/components/ClientsListContainer.tsx (hunk 2 above)
Comment (katerinamartysko) [reply src/components/ClientsListContainer.tsx]: Done

File: src/components/Search.tsx (hunk 3 above)
Comment (katerinamartysko) [reply src/components/Search.tsx:18]: Done

File: src/utils/getFirstLetters.ts (hunk 4 above)
Comment (katerinamartysko) [reply src/utils/getFirstLetters.ts]: Done

**Review by LexaLukovka — APPROVED** [review]

**Pull request merged**